### PR TITLE
[docs] Added a passage for 3D model formats.

### PR DESCRIPTION
**Pull request by dirkk0**
**Description:**

**Changes proposed:**
-
-
-

### CODE REVIEW

**Comment by donmccurdy** [comment]
+1 for having something like this in the FAQ.

Similar passage going into # 2333 — [gltf-model.md](https://github.com/aframevr/aframe/pull/2333/files#diff-65cb6350cb8b140f4af98738e4059465), perhaps we can consolidate and have one guide with more detail. May also want to link to `json-model` / `object-model` if they're going to be mentioned.

**Comment by ngokevin** [comment]
Sounds good, although the answer might be incomplete (i.e, should mention some things about glTF and the advantages/disadvantages of all). Though it's better for docs to repeat itself because people don't read every page.

File: docs/introduction/faq.md [hunk 1]
```diff
@@ -136,6 +136,10 @@ texture. There are some components in the ecosystem that enable this:
 
 - [HTML Shader][html-shader]
 
+## What 3D Model format works best?
```
Comment (cvan) [inline docs/introduction/faq.md]: can you change `What 3D Model format work` to `Which 3D model formats work`?

File: docs/introduction/faq.md [hunk 2]
```diff
@@ -136,6 +136,10 @@ texture. There are some components in the ecosystem that enable this:
...
+
+The ThreeJS JSON format works best, since it is the generic loader for the underlying ThreeJS. OBJ and DAE formats can work also, albeit with some limitations (for example, OBJ is not able to hold vectice colors).
```
Comment (cvan) [inline docs/introduction/faq.md]: for `ThreeJS JSON format`, it might be helpful to link to https://threejs.org/examples/?q=webgl_loader_json

Comment (cvan) [inline docs/introduction/faq.md]: could you change to `since it is the generic loader for the underlying ThreeJS` to `since it is the default loader`?

Comment (cvan) [inline docs/introduction/faq.md]: could you add links for these?

```markdown
[Wavefront (`.obj`)](https://en.wikipedia.org/wiki/Wavefront_.obj_file) and [COLLADA (`.dae`)](https://en.wikipedia.org/wiki/COLLADA)
```

Comment (cvan) [inline docs/introduction/faq.md]: minor typo: `vectice` should be `vertice`

**Comment by cvan** [comment]
thanks for the contribution! 👍 per @ngokevin's comments above, could you mention [`glTF`](https://en.wikipedia.org/wiki/glTF)?

**Comment by ngokevin** [comment]
> Sounds good, although the answer might be incomplete (i.e, should mention some things about glTF and the advantages/disadvantages of all).

I would mostly follow after what @donmccurdy wrote at https://aframe.io/docs/master/components/gltf-model.html#why-use-gltf

Also does not mention how to use the JSON loader.

**Comment by dirkk0** [comment]
I updated this again, not sure if this information made it. :-)

**Comment by donmccurdy** [comment]
My vote would be to go ahead and merge this, and make any tweaks to wording with followup commits.

File: docs/introduction/faq.md [hunk 3]
```diff
@@ -136,6 +136,16 @@ texture. There are some components in the ecosystem that enable this:
...
 
+## Which 3D model formats work?
+
+The [ThreeJS JSON format](https://threejs.org/examples/?q=webgl_loader_json) works best, since it is the default loader ([example](http://curious-electric.com/aframe/json/)).
+
+[Wavefront (`.obj`)](https://en.wikipedia.org/wiki/Wavefront_.obj_file) is an old, established and well known format, but has some limitations like the lack of vertice colors support ([example](http://curious-electric.com/aframe/obj/)).
+
+[COLLADA (`.dae`)](https://en.wikipedia.org/wiki/COLLADA) is an XML based format with a very rich feature set ([example](http://curious-electric.com/aframe/collada/)).
+
+In comparison to COLLADA, the supported features of [glTF (`.gltf`)](https://en.wikipedia.org/wiki/GlTF) are very similar. However, because glTF focuses on providing a “transmission format” rather than an editor format, it is more interoperable with web technologies ([example](http://curious-electric.com/aframe/gltf/), [more information](https://aframe.io/docs/master/components/gltf-model.html#why-use-gltf)).
```
Comment (cvan) [inline docs/introduction/faq.md]: I would link to https://aframe.io/aframe/examples/primitives/models/ and https://aframe.io/aframe/examples/test/model/

**Comment by dirkk0** [comment]
ok, I took another stab on on it.

**Comment by ngokevin** [comment]
Tweaked a bit. Merged, thanks!

**Pull request merged**